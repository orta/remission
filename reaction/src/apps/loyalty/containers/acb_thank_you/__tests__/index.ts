import * as ReactTestUtils from "react-dom/test-utils"
import * as renderer from "react-test-renderer"

import AcbThankYou from "../index"

describe("ACB Thank You", () => {
  it("renders the snapshot", () => {
    const thankYou = renderer.create(AcbThankYou({}))
    expect(thankYou).toMatchSnapshot()
  })

  it("renders the correct content", () => {
    const thankYou = ReactTestUtils.renderIntoDocument(AcbThankYou({})) as any
    const pTags = ReactTestUtils.scryRenderedDOMComponentsWithTag(thankYou, "p")
    const earlyAccessTag = pTags.find(tag => tag.textContent === "EARLY ACCESS")
    expect(earlyAccessTag).toBeTruthy()
    const divTags = ReactTestUtils.scryRenderedDOMComponentsWithTag(thankYou, "div")
    const titleTag = divTags.find(tag => tag.textContent === "Artsy Collector Loyalty Program")
    expect(titleTag).toBeTruthy()
  })
})
