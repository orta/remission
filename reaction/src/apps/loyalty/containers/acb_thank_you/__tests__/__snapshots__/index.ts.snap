// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ACB Thank You renders the snapshot 1`] = `
<div
  className="file__Container-s1k4n81i-0 hGpafc"
>
  <div
    className="file__Nav-s1w500ep-0 krvdox"
  >
    <a
      className="file__NavIcon-s1w500ep-1 eNnazU"
      href={undefined}
    >
      <div
        className="file-s1ekx40u-0 krLqUx"
        style={undefined}
      >
        
      </div>
    </a>
    <a
      className="file__StyledLink-mgasw5-0 dIJcvg"
      href="/"
    >
      Back To Artsy
    </a>
  </div>
  <div
    className="file__Section-s1k4n81i-1 corYNg"
  >
    <p
      className="file__Text-s4rtu63-0 btgjDp"
    >
      EARLY ACCESS
    </p>
  </div>
  <div
    className="file__StyledTitle-s3pz4jh-0 izQygJ"
  >
    Artsy Collector Loyalty Program
  </div>
  <div
    className="file__Section-s1k4n81i-1 corYNg"
  >
    <p
      className="file__Text-s4rtu63-0 eEhqjQ"
    >
      Thank you for your interest in the program.
      <br />
      Have questions? Get in touch:  
      <a
        className="file__StyledTextLink-s18afosz-0 bJfHCj"
        href="mailto:loyalty@artsy.net"
      >
        loyalty@artsy.net
      </a>
    </p>
  </div>
  <div
    className="file__ButtonWrapper-s1k4n81i-2 jFIzwE"
  >
    <a
      className="file__InvertedButton-s2qkmvo-0 MNMDa file__StyledButton-muis26-0 exlWFO"
      href="/"
    >
      <span>
        Back to Artsy
      </span>
    </a>
  </div>
</div>
`;
